js/privatebin.js

```diff
@@ -930,7 +930,7 @@ jQuery.PrivateBin = (function($) {
             }
 
             // load strings from JSON
-            $.getJSON('i18n/' + newLanguage + '.json', function(data) {
+            $.getJSON('i18n/' + newLanguage + '.json?v=' + Math.floor(Date.now() / 60000), function(data) {
```

Comment:
We could actually use the version of the privatebin.js script, to avoid invalidating the cache every day (1 day = 84200s ~= 60000s).
```suggestion
            $.getJSON('i18n/' + newLanguage + '.json?' + new URL(document.currentScript.src).search.slice(1), function(data) {
```

Reply:
Hi, it's done